src/service/resources-impl.js

```diff
@@ -2723,5 +2716,24 @@ export let SizeDef;
  * @param {!./ampdoc-impl.AmpDoc} ampdoc
  */
 export function installResourcesServiceForDoc(ampdoc) {
-  registerServiceBuilderForDoc(ampdoc, 'resources', Resources);
+  registerServiceBuilderForDoc(ampdoc, 'resources', getResources);
+}
+
+/**
+ * @param {!./ampdoc-impl.AmpDoc} ampdoc
+ */
+export function installOwnersServiceForDoc(ampdoc) {
+  registerServiceBuilderForDoc(ampdoc, 'owners', getResources);
```

Comment:
Sorry for missing context here. Why split resources into three services? 

Will non-A4A return the same object instance for the three services (resources, owners, mutator)?

Reply:
This is after a discussion with @dvoytenko . It aligns with the goal of the Bento initiative to split this big services into smaller ones. Some of them are core rendering critical (`resources`), some are not (`mutators`). `owners` should finally go away after `layers` landing (@jridgewell) 

As you can see in this PR, we return same instance for those services in regular AMP. But eventually, they should be split up (as part of the Bento project).

Let me know if this all makes sense to you.

Reply:
Sounds good.